schema/home/results.js

```diff
@@ -43,16 +43,14 @@ const moduleResults = {
       return slice(shuffle(artworks), 0, RESULTS_SIZE);
     });
   },
-  saved_works: ({ accessToken }) => {
-    return gravity.with(accessToken)('me').then((user) => {
-      return gravity
-        .with(accessToken)('collection/saved-artwork/artworks', {
-          size: RESULTS_SIZE,
-          user_id: user.id,
-          private: true,
-          sort: '-position',
-        });
-    });
+  saved_works: ({ accessToken, userID }) => {
```

Comment:
I think this is a breaking change for web if deployed, until the id is sent in headers?